plugins/jdg72/pom.xml

```diff
@@ -26,24 +26,24 @@
             </property>
          </activation>
          <properties>
-            <version.infinispan>8.5.0.ER3-redhat-9</version.infinispan>
+            <version.infinispan>8.5.0.ER3-redhat-10</version.infinispan>
```

Comment:
Ehmm, did you mean 8.5.0.ER4-redhat-10 ?

Reply:
Fixed.